src/components/jogging/JoggingStore.ts

```diff
@@ -88,7 +97,9 @@ export class JoggingStore {
   selectedCartesianMotionType: "translate" | "rotate" = "translate"
 
   /** True when the API is busy doing a planned increment jog motion */
-  incrementJoggingInProgress = false
+  incrementJogging: IncrementalJoggingState = {
+    jogging: undefined,
```

Comment:
why the wrapper? I'd just do `currentIncrementJogging: IncrementJoggingState|null = null`

also the comment above is wrong now 😄

Reply:
Have to check again but mobx did not track objects with undefined/null but maybe it was only undefined as init value

Reply:
it's only undefined yep 😄